rllib/utils/replay_buffers/prioritized_episode_replay_buffer.py

```diff
@@ -16,6 +16,104 @@
 
 
 class PrioritizedEpisodeReplayBuffer(EpisodeReplayBuffer):
+    """Prioritized Replay Buffer that stores episodes by their ID.
+
+    This replay buffer stores episode data (more specifically `EpisodeV2`
```

Comment:
nit: `SingleAgentEpisode` objects.

EpisodeV2 is wrong here. It's an old-API-stack class.